app/models/dto.py

```
@@ -63,6 +63,42 @@ class AvailabilityRequest(BaseModel):
     start_hour: str = Field(..., description="Start time (HH:MM)")
     end_hour: str = Field(..., description="End time (HH:MM)")
     
+    @field_validator('start_hour', 'end_hour')
+    @classmethod
+    def validate_hour_format(cls, v: str) -> str:
+        if not re.match(r"^(0[0-9]|1[0-9]|2[0-4]):00$", v):
+            raise ValueError(f"시간은 'HH:00' 포맷(정각)으로 입력해야 합니다. (잘못된 입력: {v})")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)